source/features/indented-code-wrapping.css

```diff
@@ -0,0 +1,13 @@
+.rgh-softwrapped-code .blob-code:not(.blob-code-hunk) {
+	line-height: 21px; /* Compensation for block-ifying table cells */
+}
+
+.rgh-code-wrapping-enabled .blob-code-hunk, /* Wrap code eagerly to avoid layout changes later */
+.rgh-softwrapped-code .blob-code-hunk {
+	white-space: pre-wrap;
+}
+
+.rgh-softwrapped-code tr:not(.js-expandable-line) .blob-code-inner:not(.blob-code-hunk) {
+	display: block;
```

Comment:
Since the height change is caused by this, shouldn’t `line-height` be here?

Reply:
Because that affects `td` inside suggestion blocks, changing their height from `26px` to `29px` if `line-height` is added.

We only need to add the CSS for `td.blob-code` inside diff tables, which already have `line-height` of `20px` (now changed to `21px`).

This is evident from the GIF above, the suggestion box is not affected before this change (or after for that matter).